src/eip7594/eip7594.c

```diff
@@ -159,6 +176,7 @@ C_KZG_RET compute_cells_and_kzg_proofs(
  * @param[in]   num_cells           The number of available cells provided
  * @param[in]   s                   The trusted setup
  *
+ * @remark At least 50% of CELLS_PER_EXT_BLOB cells must be provided.
```

Comment:
Moreover the current code actually has the expansion rate `2` hardcoded in several places as an integer rather than as a named constant. If we need the code scalable to other rates, then the constant should be named.

Reply:
Good idea. I'll make an `EXPANSION_FACTOR` constant.

Reply:
Also decided to make a `CELLS_PER_BLOB` constant.